client/reader/stream/reader-tag-sidebar/index.jsx

```diff
@@ -0,0 +1,83 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import TagLink from 'calypso/blocks/reader-post-card/tag-link';
+import formatNumberCompact from 'calypso/lib/format-number-compact';
+import ReaderListFollowingItem from 'calypso/reader/stream/reader-list-followed-sites/item';
+import '../style.scss';
+
+class ReaderTagSidebar extends Component {
+	static defaultProps = {
+		sitesPerPage: 25,
+		trendingTags: [],
+		relatedSites: [],
+	};
+
+	static propTypes = {
+		trendingTags: PropTypes.array,
+		relatedSites: PropTypes.array,
+		postCount: PropTypes.number,
+		authorCount: PropTypes.number,
+		sitesPerPage: PropTypes.number,
```

Comment:
nit: I don't think the sitesPerPage property is ever passed in, maybe it should just be a `const` for now